### PR TITLE
Move from doctests to unittests and split them up

### PR DESCRIPTION
**Pull request by mauritsvanrees**
It may be hard to review the relatively big diff in one go, but viewing the individual commits should make the changes really clear.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
See issue #12.

File: plone/app/redirector/tests/test_browser.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        self.browser.getControl('New Title').value = 'Bar'
+        self.browser.getControl('Rename').click()
+        self.assertListEqual(list(storage), ['/plone/foo'])
+        self.assertEqual(storage.get('/plone/foo'), '/plone/bar')
+
+
+def test_suite():
+    suite = unittest.TestSuite()
+    suite.addTest(unittest.makeSuite(TestBrowser))
+    return suite
```
Comment (gforcada) [inline plone/app/redirector/tests/test_browser.py]: is this still needed? I thought that this is from a by-gone era where test runners where simpler, but nowadays they pick them up easily :thinking:

File: plone/app/redirector/tests/test_storage.py [hunk 2]
```diff
@@ -1,13 +1,137 @@
...
+        # And same for the back references.
+        self.assertListEqual(p.redirects('first'), ['second', 'third'])
+        self.assertListEqual(p.redirects('second'), [])
+        self.assertListEqual(p.redirects('third'), [])
 
 
 def test_suite():
     suite = unittest.TestSuite()
-    suite.addTest(doctest.DocTestSuite(storage, optionflags=optionflags))
+    suite.addTest(unittest.makeSuite(TestStorage))
```
Comment (gforcada) [inline plone/app/redirector/tests/test_storage.py]: ditto as before, is this still needed? :thinking:

**Pull request merged**